### ISSUE TITLE
데이터 전처리 및 특성 공학

### ISSUE THREAD

**Issue by GAEUN0914** [GAEUN0914/SCNU-SW#6]
import pandas as pd
import numpy as np

def create_technical_indicators(df: pd.DataFrame) -> pd.DataFrame:
    """
    이동평균선 (MA) 및 상대강도지수 (RSI)를 계산하여 Feature를 추가합니다.
    """
    # 5일 및 20일 이동평균선 (MA)
    df['MA_5'] = df['Close'].rolling(window=5).mean()
    df['MA_20'] = df['Close'].rolling(window=20).mean()
    
    # 일별 수익률
    df['Daily_Return'] = df['Close'].pct_change()
    
    # 상대강도지수 (RSI) 계산
    # (간단한 구현, 라이브러리(ta-lib) 사용 권장)
    delta = df['Close'].diff()
    gain = delta.where(delta > 0, 0)
    loss = -delta.where(delta < 0, 0)
    avg_gain = gain.rolling(window=14).mean()
    avg_loss = loss.rolling(window=14).mean()
    RS = avg_gain / avg_loss
    df['RSI'] = 100 - (100 / (1 + RS))
    
    return df.dropna()

if __name__ == '__main__':
    # 예시 데이터 로드 (실제로는 data_loader.py 사용)
    data = {'Close': [100, 101, 102, 103, 104, 105, 104, 103, 102, 101]} * 5
    df = pd.DataFrame(data)
    df_features = create_technical_indicators(df)
    print(df_features.tail())
